examples/minimq-stm32h7/src/main.rs

```diff
@@ -43,8 +42,8 @@ static mut NET_STORE: NetStorage = NetStorage {
 static mut DES_RING: ethernet::DesRing = ethernet::DesRing::new();
 
 #[derive(Serialize, Deserialize)]
-struct Temperature {
-    temperature_c: f32,
+struct Random {
+    random: Option<u32>
```

Comment:
Why is it an Optional param? Shouldn't it just be a u32?

Reply:
The RNG returns a Result (not sure under which conditions it can fail), but the error variant of the Result wasn't serializable. I decided to just transform it into an Option that will return None in the case of failure.